src/amo/components/AddonsCard/index.js

```diff
@@ -34,13 +34,24 @@ type Props = {|
 
 export default class AddonsCard extends React.Component<Props> {
   cardContainer: React.ElementRef<any> | null;
+  keyCount: number;
 
   static defaultProps = {
     editing: false,
     loading: false,
     placeholderCount: DEFAULT_API_PAGE_SIZE,
   }
 
+  constructor(props: Props) {
+    super(props);
+
+    this.keyCount = 0;
+  }
+
+  getKey = () => {
+    return this.keyCount++;
```

Comment:
This could unfortunately lead to SSR issues if client and server don't come up with the same values for some reason. It would be better to have fixed keys.